docs/05-infrastructure-nodes.md

```diff
@@ -0,0 +1,318 @@
+infrastructure components at no additional charge. In other words, a node
+that is only running core OpenShift infrastructure components is not counted
+in terms of the total number of subscriptions required to cover the
+environment.
+
+OpenShift components that fall into the infrastructure categorization
+include:
+
+* kubernetes and OpenShift control plane services ("masters")
+* router
+* container image registry
+* cluster metrics collection
+* cluster aggregated logging
+* service brokers
+
+Any node running a container/pod/component not described above is considered
+a worker and must be covered by a subscription.
+
+## More MachineSet Details
+In the cluster scaling exercises you explored the use of `MachineSets` and
+adding replicas within them. In the case of an infrastructure node, we want
+to create additional `Machines` that have specific kubernetes labels. Then,
+we can tell the various components to run specifically on hosts with those
+labels.
+
+To accomplish this, you will create additional `MachineSets`. The easiest way
+to do this is to `get` the existing `MachineSets` into a file, and then
+modify them. This is because the `MachineSet` has some details that are
+specific to the AWS region that the cluster is deployed in, like the AWS EC2
+AMI ID. For example, given the following output of `oc get machineset -n
+openshift-cluster-api`:
+
+    NAME                         DESIRED   CURRENT   READY     AVAILABLE   AGE
+    190125-3-worker-us-west-1b   2         2         2         2           3h
+    190125-3-worker-us-west-1c   1         1         1         1           3 
+
+There are two available EC2 AZs into which we can deposit infrastructure
+components (`1b` and `1c`). Take a look at one specifically with `oc get
+machineset 190125-3-worker-us-west-1b -n openshift-cluster-api -o yaml`.
+There are a few very important sections.
+
+### Metadata
+The `metadata` on the `MachineSet` itself includes information like the name
+of the `MachineSet` and various labels:
+
+```YAML
+metadata:
+  creationTimestamp: 2019-01-25T16:00:34Z
+  generation: 1
+  labels:
+    sigs.k8s.io/cluster-api-cluster: 190125-3
+    sigs.k8s.io/cluster-api-machine-role: worker
+    sigs.k8s.io/cluster-api-machine-type: worker
+  name: 190125-3-worker-us-west-1b
+  namespace: openshift-cluster-api
+  resourceVersion: "9027"
+  selfLink: /apis/cluster.k8s.io/v1alpha1/namespaces/openshift-cluster-api/machinesets/190125-3-worker-us-west-1b
+  uid: 591b4d06-20ba-11e9-a880-068acb199400
+```
+
+### Selector
+The `MachineSet` defines how to create `Machines`, and the `Selector` tells
+the operator which machines are associated with the set:
+
+```YAML
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      sigs.k8s.io/cluster-api-cluster: 190125-3
+      sigs.k8s.io/cluster-api-machineset: 190125-3-worker-us-west-1b
+```
+
+In this case, the cluster name/ID is `190125-3` and there is an additional
```

Comment:
nit: That's the cluster name (the cluster ID is a UUID), so probably drop "/ID".

Reply:
Might want to pick a more "name-like" name maybe ocp-training instead of a string of numbers.

Reply:
fixed (name/id).
I'm not particularly concerned with "friendly" names. Remember, clusters are cattle, not pets! :)